src/modules/pos/validator/__fixtures__/mockRewardsLocked.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-lines, import/prefer-default-export */
```

Comment:
@oskarleonard I believe the max-lines rule wasn't broken in this file and hence no need to disable it

Reply:
True, copy paste error